server/src/middleware/users/users-middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { RequestHandler } from 'express';
```

Comment:
Use dataValidationMiddleware for body validation. Check out how it is used with clan api